string name  
uint8 product_type  
uint32 price  

uint8 PRODUCT_TYPE_GIN = 0  
uint8 PRODUCT_TYPE_WHISKEY = 1  
uint8 PRODUCT_TYPE_VODKA = 2